### DISCUSSION TITLE
next/script package with custom document

### DISCUSSION THREAD

**Discussion by ch3rn1k** [vercel/next.js#28885] [Help] [answered]
### What version of Next.js are you using?

11.1.2

### What version of Node.js are you using?

14.17.0

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### How are you deploying your application?

next start

### Describe the Bug

No way to use `next/script` optimization package inside custom `_document.js`

### Expected Behavior

Expected to initialize once metric scripts inside custom '_document.js' file.

### To Reproduce

Try to add any script with `next/script` inside `_document.js`

**Comment by ch3rn1k** [comment]
Well, it's not a bug so, but how to optimize metric scripts using this package? For sure it needs to be initialized only once...

```
next/script must not be placed in either a next/head component or in pages/_document.js.
```

**Comment by stefanprobst** [comment, marked as answer]
you can put the `next/script` component in `pages/_app.js`.
